src/test/kotlin/uk/gov/justice/digital/assessments/config/ClientTrackingConfigurationTest.kt

```diff
@@ -1,92 +0,0 @@
-package uk.gov.justice.digital.assessments.config
```

Comment:
we have got examples of how to do this with latest spring boot 3 if needed

Reply:
Updated